Dockerfile

```diff
@@ -58,3 +58,14 @@ RUN patch /usr/local/bin/docker-entrypoint.sh docker-entrypoint.sh.patch
 RUN mkdir -p /etc/mongo-key && chown mongodb:mongodb /etc/mongo-key
 
 CMD ["--replSet", "rs", "--keyFile", "/etc/mongo-key/mongodb.key"]
+
```

Comment:
Do still need this after implementing https://github.com/thenewboston-developers/Node/pull/64/files#r783295831 comment?